plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreRoleManager.java

```diff
@@ -147,6 +148,12 @@ private void addRoleOptions(StringBuilder script, PostgreRole role, NestedObject
             options.append(" PASSWORD ").append("'").append(role.getDataSource().getSQLDialect().escapeString(role.getPassword())).append("'");
             command.setDisableSessionLogging(true); // Hide password from Query Manager
         }
+
+        if (role.getValidUntil() != null) {
+            Timestamp timestamp = Timestamp.valueOf(role.getValidUntil());
+            options.append(" VALID UNTIL ").append(SQLUtils.quoteString(role, timestamp.toString()));
```

Comment:
We just convert timestamp to string. But is Timestamp conversion is compatible with PostgreSQL server format?